tests/_methods/test_method_helpers.py

```diff
@@ -38,6 +41,7 @@ def test_list_methods_private() -> None:
         "densecl",
         "dino",
         "distillation",
+        "distillationv2",
         "simclr",
```

Comment:
I think `distillationv1` must also be returned from here. Otherwise you cannot set `method="distillationv1"` when training.